packages/perseus/src/widgets/interactive-graphs/graphs/polygon.tsx

```diff
@@ -153,6 +158,218 @@ export const PolygonGraph = (props: Props) => {
     );
 };
 
+export const UnlimitedPolygonGraph = (props: Props) => {
+    const [hovered, setHovered] = React.useState(false);
+    // This is more so required for the re-rendering that occurs when state
+    // updates; specifically with regard to line weighting and polygon focus.
+    const [focusVisible, setFocusVisible] = React.useState(false);
+
+    const {dispatch} = props;
+    const {
+        coords,
+        showAngles,
+        showSides,
+        range,
+        snapStep,
+        snapTo = "grid",
+    } = props.graphState;
+
+    const graphState = useGraphConfig();
+
+    // TODO [catjohnson]: Explore abstracting this code as it is similar to point.tsx
+    // and hopefully we can cut down ont the unlimited graph redundancy.
+    const {
+        range: [[minX, maxX], [minY, maxY]],
+        disableKeyboardInteraction,
+    } = graphState;
+
+    const width = maxX - minX;
+    const height = maxY - minY;
+    const [[widthPx, heightPx]] = useTransformDimensionsToPixels([
+        width,
+        height,
+    ]);
```

Comment:
This also doing some computation to transform the dimensions to pixels and I see we use this in circle.tsx. Should I not be using this function?

Reply:
`useTransformDimensionsToPixels` is intended for cases where we have some dimensions in graph units and we want to convert them to pixels. Here we don't need to convert units, because the pixel dimensions [are available already](https://github.com/Khan/perseus/blob/f5425ce3eda6d3294919e35f99e08fd6d346005e/packages/perseus/src/widgets/interactive-graphs/reducer/use-graph-config.ts#L15-L16) on the graph config.

Both ways of doing it are correct, but getting the pixel dimensions that are already computed is simpler and probably slightly faster, so it seems to me like the better option.